core/arch/arm/include/arm.h

```diff
@@ -45,6 +45,7 @@
 #define MPIDR_CLUSTER_SHIFT	MPIDR_AFF1_SHIFT
 #define MPIDR_CLUSTER_MASK	MPIDR_AFF1_MASK
 
+#define MPIDR_AARCH32_AFF_MASK	0xffffff
```

Comment:
add empty line below + GENMASK_32(31, 0)

Reply:
I'll add that and compose this define from MPIDR_AFF0_MASK..MPIDR_AFF2_MASK instead.